css/styles.css

```diff
@@ -1,13 +1,126 @@
 *, *::before, *::after {
-  box-sizing: border-box;
+    box-sizing: border-box;
 }
 
 body {
     margin: 0;
     padding: 0;
+    background-image: radial-gradient(#2E4CF5, #4B91F2);
+    font-family: 'Special Elite', cursive;
 }
 
 img {
-  width: 100%;
-  height: auto;
-}
\ No newline at end of file
+    width: 100%;
+    height: auto;
+}
+
+header {
+    margin: 4rem 2rem 2rem;
+}
+
+h1 {
+    margin-top: 0.5rem;
+    text-align: center;
+
+}
+
+.collection {
+    width: 95vw;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-items: center;
+    gap: 0.5rem;
+    text-align: center;
+}
+
+.deck-card {
+    width: 95%;
+    margin-bottom: 1rem;
+    border: 0.2rem solid #000000;
+    border-radius: 0.2rem;
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
Same deal here, don't think you need this line

Reply:
Removed